src/mk.sh

```diff
@@ -489,6 +503,26 @@ function kernel_deploy
   fi
 }
 
+# This function retrieves and prints information related to the building
+# process of the kernel.
+function build_info()
+{
+  if [[ -f ".config" ]]; then
+    local kernel_name=$(get_kernel_release "$flag")
+    local kernel_version=$(get_kernel_version "$flag")
+    local compiled_modules=$(grep -c "=m" .config)
+
+    say "Kernel source information"
+    echo -e "\tName: $kernel_name"
+    echo -e "\tVersion: $kernel_version"
```

Comment:
This should be outside of this condition. The same is true for `local kernel_version=$(get_kernel_version "$flag")` and `local kernel_name=$(get_kernel_release "$flag")`. The only thing that needs to be inside this function is the operations that deal with `.config`.